### DISCUSSION TITLE
Laser cuts lines twice

### DISCUSSION THREAD

**Discussion by Luna-gorka** [arkypita/LaserGRBL#1540] [Q&A]
My laser cuts every line 2 times. It finishes the line then repeats. I use Inkscape PNG images.  1st cut line is perfect but the second cut Line is wobbly 
Line Width 1px. 
 My image does not have a double line. I am new to engraving and can't find the problem.

**Comment by Tribo-2dot0** [comment]
If the drawing is with lines and you choose Vectorize, it will be double lined (one inside and one outside the original line)... try using Centerline which will be just one line.

**Reply by nDAWNm** [reply]
Ever since the "upgrade" of LaserGRBL, I have been having everything goof up.  It cuts double lines on everything and now it has decided to take a square object and round off the corners.  I use, and have always used, Inkscape first. What is going on?

Is there a way to go back to the original LaserGRBL?

**Reply by arkypita** [reply]
LaserGRBL has never changed how the image are processed since its first version, so if you find difference is not because the "upgrade". If you don' t trust feel free to download any previous version:
https://github.com/arkypita/LaserGRBL/releases

**Reply by arkypita** [reply]
https://lasergrbl.com/usage/raster-image-import/vectorization-tool/

**Comment by Luna-gorka** [comment]
Thank you. I also found out that the picture must be unioned inkscape.  It
is working now. You help is much appreciated.

**Reply by nDAWNm** [reply]
What is "unioned" inkscape?

**Comment by nDAWNm** [comment]
Ok, thanks!
Can you tell me what I'm doing wrong or how to get it to stop doing that? I noticed others are having the same issue.

Nancy

On November 23, 2024, at 3:50 AM, arkypita ***@***.***> wrote:

LaserGRBL has never changed how the image are processed since its first version, so if you find difference is not because the "upgrade". If you don' t trust feel free to download any previous version:
https://github.com/arkypita/LaserGRBL/releases

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.￼Message ID: ***@***.***>

[ { ***@***.***": "<a href="http://schema.org">http://schema.org</a>", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "<a href="https://github.com/arkypita/LaserGRBL/discussions/1540#discussioncomment-11356485">https://github.com/arkypita/LaserGRBL/discussions/1540#discussioncomment-11356485</a>", "url": "<a href="https://github.com/arkypita/LaserGRBL/discussions/1540#discussioncomment-11356485">https://github.com/arkypita/LaserGRBL/discussions/1540#discussioncomment-11356485</a>", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "<a href="https://github.com">https://github.com</a>" } } ]
